### PR TITLE
Fix bug 1148727 - Reenable user after removing a ban.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: kuma/users/admin.py [hunk 1]
```diff
@@ -21,7 +21,8 @@ class UserAdmin(BaseUserAdmin):
 
 class UserBanAdmin(admin.ModelAdmin):
     fields = ('user', 'by', 'reason', 'is_active')
-    list_display = ('user', 'by', 'reason')
+    list_display = ('user', 'by', 'reason', 'is_active')
+    list_editable = ('is_active',)
```
Comment (jezdez) [inline kuma/users/admin.py]: This also adds the ability to toggle a ban easily from the admin list.

Comment (robhudson) [inline kuma/users/admin.py]: Having both `UserBan.is_active` and `User.is_active` that are essentially opposites of each other is very confusing here.

File: kuma/users/tests/test_models.py [hunk 2]
```diff
@@ -108,10 +108,19 @@ def test_ban_user(self):
...
         ok_(testuser_unbanned.is_active)
+
+        ban.is_active = True
+        ban.save()
+        testuser_unbanned = self.user_model.objects.get(username='testuser')
+        ok_(not testuser_unbanned.is_active)
+
+        ban.delete()
+        testuser_unbanned = self.user_model.objects.get(username='testuser')
+        ok_(testuser_unbanned.is_active)
```
Comment (robhudson) [inline kuma/users/tests/test_models.py]: Seems like a good idea to also assert the `is_banned` and `active_ban` methods here as well.

**Comment by robhudson** [comment]
r+wc

File: kuma/users/admin.py (hunk 1 above)
Comment (kleberveridiano) [inline kuma/users/admin.py]: ok!

File: kuma/users/tests/test_models.py (hunk 2 above)
Comment (kleberveridiano) [inline kuma/users/tests/test_models.py]: ok!

Comment (jezdez) [inline kuma/users/tests/test_models.py]: Good idea!

File: kuma/users/admin.py (hunk 1 above)
Comment (jezdez) [inline kuma/users/admin.py]: Yeah, I don't like it either, should we rename it?

Comment (robhudson) [inline kuma/users/admin.py]: I don't think it is necessary for this PR but it was a cause of major confusion for me at first glance.

Comment (jezdez) [inline kuma/users/admin.py]: Understood, I'll file a ticket

Comment (jezdez) [inline kuma/users/admin.py]: https://bugzilla.mozilla.org/show_bug.cgi?id=1178399

**Pull request merged**